kars/0003-gpu-sharing/README.md

```diff
@@ -0,0 +1,40 @@
+# KAR-0003: GPU Sharing
+
+## Description
+
+If accelerator supports GPU sharing, provide well-defined mechanisms for GPU sharing to improve utilization for workloads that do not require a full dedicated GPU. If hardware-level partitioning is supported, these fractional GPU resources should be exposed as distinct schedulable resources via DRA once it’s supported. If software-based sharing (e.g. time-slicing) is supported, device plugin should be used to allow oversubscription of GPUs, with a forward-looking path to DRA.
```

Comment:
> If hardware-level partitioning is supported, these fractional GPU resources should be exposed as distinct schedulable resources via DRA once it’s supported.

Is "once it's supported" referring to the [partitionable devices](https://github.com/kubernetes/enhancements/issues/4815) feature currently being alpha, and "supported" coincides with the feature reaching GA? So the requirement is that drivers utilize that feature when it graduates to GA? [Consumable capacity](https://github.com/kubernetes/enhancements/issues/5075) also seems like a valid way to fulfill this, even if none of the GPU vendors can currently model their GPUs that way.